tests/testthat/test-carehomes-check.R

```diff
@@ -953,3 +953,24 @@ test_that("Individuals can infect in compartment with non-zero transmission", {
   helper("ICU_transmission", "ICU_W_R_unconf", "gamma_ICU_W_R_step")
   helper("ICU_transmission", "ICU_W_R_conf", "gamma_ICU_W_R_step")
 })
+
+
+test_that("No one is hospitalised, no-one recovers in edge case", {
```

Comment:
same comment as in the previous PR i.e. could we also add a test which checks this when setting the probabilities of death to zero ONLY for strain 2 using the strain_rel_severity parameter?